raiden/assetmanager.py

```diff
@@ -369,14 +375,14 @@ def get_best_routes(self, amount, target, lock_timeout=None):
                     )
                 continue
 
-            # Our partner won't accept a locked transfer that can expire after
-            # the settlement period, otherwise the secret could be revealed
-            # after channel is settled and he would lose the asset, or before
-            # the minimum required.
-            valid_timeout = channel.reveal_timeout <= lock_timeout < channel.settle_timeout
+            if lock_timeout:
+                # Our partner won't accept a locked transfer that can expire after
+                # the settlement period, otherwise the secret could be revealed
+                # after channel is settled and he would lose the asset, or before
```

Comment:
Since you moved this comment can we also make some changes to it?
`after channel is settled` -> `after the channel is settled`